argocd/federated-etl-argo/argocd-kubecost-primary.yaml

```
@@ -0,0 +1,45 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: kubecost
+  namespace: argocd
+spec:
+  project: default
+  source:
+    chart: cost-analyzer
+    repoURL: https://kubecost.github.io/cost-analyzer/
+    targetRevision: 1.105.7
+    helm:
+      releaseName: kubecost
+      valueFiles:
+      # Be sure to set the relative path to the secondary values file kubecost-primary-federator-values.yaml
```

Nit: Should this be `primary`?
```suggestion
      # Be sure to set the relative path to the primary values file kubecost-primary-federator-values.yaml
```